eDNAExplorer_Alpha.R

```diff
@@ -133,6 +131,6 @@ function(First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,F
       theme_bw()+geom_point()+geom_smooth()
   }
   #Save plot as json object
-  jfig <- plotly:::to_JSON(ggplotly(p))
+  jfig <- plotly_json(p, FALSE)
```

Comment:
which one is correct?

Reply:
The one with the final line 'jfig <- plotly_json(p, FALSE)'

Reply:
Ok so no changed needed to this PR